frontend/src/modules/processes/actions.js

```diff
@@ -0,0 +1,26 @@
+import { listProcesses, listPropertyValues } from "../experimentRuns/actions";
+
+export const listProcessProperties = (id) => async (dispatch, getState) => {
+    if (getState().propertyValues.isFetching)
+        return;
+
+    const { itemsByID: processesByID } = getState().processes    
+    const { itemsByID: propertyValuesByID } = getState().propertyValues
+
+    if (!(id in processesByID)) {
+        return await dispatch(listProcesses({ id__in: id }))
```

Comment:
I don't think you want to `return` here. You just want to wait until the processes have been loaded and then continue to the next step.